# Given a set of visual features, estimate a pose
# TODO: pose with covariance
VisualFeature[] visualFeatures
---
geometry_msgs/Pose pose